### PR TITLE
authentication 

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Features added in this PR -
- Navbar
- Sidebar UI
- Login Page
- Signup Page
- Login feature
- Logout feature

As this is the First PR it has many Files ,
Here are the Files to look for : 

- Components : `src/Components` (Navbar and Sidebar)
- authContext : `src/Context/Auth/auth-context.js`
- utils : `src/utils` (login,logout)
- Pages : `src/Pages` (Login , SignUp and Notes)

### CODE REVIEW

File: src/Pages/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+              onClick={() => loginHandler(user, setAuth, navigate)}
+            >
+              Login
+            </button>
+            <button
+              class="login-btn login-guest-btn"
+              onClick={(e) => {
+                e.preventDefault();
+                loginHandler(guestLogin, setAuth, navigate);
+                console.log("login state is ", auth.isLogIn);
```
Comment (aiman-ansari) [inline src/Pages/Login/Login.jsx]: suggestion: remove console log

File: src/Pages/Notes/NotesPage.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Navbar } from "../../components/Navbar/Navbar";
+import { Sidebar } from "../../components/Sidebar/Sidebar";
```
Comment (aiman-ansari) [inline src/Pages/Notes/NotesPage.jsx:2]: Suggestion : you can create index.js file in component folder and import these from there

File: src/Pages/Signup/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  return (
+    <div>
+      <Navbar />
+      <div class="login-wrapper">
+        <div class="login">
+          <h1>SignUp</h1>
+          <div class="login-credentials">
+            <div class="user-name">
+              <div class="first-name">
+                <p for="">First Name</p>
```
Comment (aiman-ansari) [inline src/Pages/Signup/Signup.jsx]: why you use for inside p tag?

File: src/Utils/signupHandler.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+const signupHandler = async (firstN, lastN, email, pswrd) => {
```
Comment (aiman-ansari) [inline src/Utils/signupHandler.js]: naming could be improve here

**Review by aiman-ansari — APPROVED** [review]
Good folder structure , neat and clean code .
I like the way how you break everything into small components.
Some suggestions are given do check

File: src/Pages/Login/Login.jsx (hunk 1 above)
Comment (atharvabhanage02) [reply src/Pages/Login/Login.jsx]: Done

File: src/Pages/Signup/Signup.jsx (hunk 3 above)
Comment (atharvabhanage02) [reply src/Pages/Signup/Signup.jsx]: I did it by mistake . Removed it .

**Review by paarthbajaj — APPROVED** [review]
Code looks good. Just one suggestion, I observed repetitive CSS code which you can reduce by making utility classes. Rest is good. Keep it up!

**Pull request merged**